pihole_adlist_tool

```diff
@@ -833,26 +837,26 @@ if [ "$FURTHER_ACTION" -eq 3 ]; then
     # remove all domains from gravity_dup that are also contained in that adlist
     # count how many domains are still on gravity_dup
 
-    adlist_conf_minimal_enabled=(`sqlite $TEMP_DB "select id from adlist where unique_domains_covered IS NOT NULL;"`)
+    mapfile -t adlist_conf_minimal_enabled < <(sqlite $TEMP_DB "select id from adlist where unique_domains_covered IS NOT NULL;")
 
     sqlite $TEMP_DB "CREATE TABLE gravity_dup AS SELECT * FROM gravity_strip"
 
     for adlist_id in "${adlist_conf_minimal_enabled[@]}"; do
        sqlite $TEMP_DB "DELETE FROM gravity_dup WHERE domain IN (SELECT domain from gravity_dup where adlist_id=$adlist_id);"
     done
 
-    LEFT_DOMAINS=(`sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;"`)
+    mapfile -t LEFT_DOMAINS < <(sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;")
 
-    while [[ $LEFT_DOMAINS != [0] ]]; do
-        current_id=(`sqlite $TEMP_DB "Select adlist_id from gravity_dup group by adlist_id order by count (domain) desc, adlist_id asc limit 1;"`);
+    while [[ "${LEFT_DOMAINS[*]}" != [0] ]]; do
+        mapfile -t current_id < <(sqlite $TEMP_DB "Select adlist_id from gravity_dup group by adlist_id order by count (domain) desc, adlist_id asc limit 1;");
```

Comment:
`current_id` should be a single number, no array

Reply:
My local `shellcheck` complained:
```
In pihole_adlist_tool line 850:
    while [[ $LEFT_DOMAINS != [0] ]]; do
             ^-----------^ SC2128: Expanding an array without an index only gives the first element.


In pihole_adlist_tool line 851:
    current_id=(`sqlite $TEMP_DB "Select adlist_id from gravity_dup group by adlist_id order by count (domain) desc, adlist_id asc limit 1;"`);
                ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
                ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean:
    current_id=($(sqlite $TEMP_DB "Select adlist_id from gravity_dup group by adlist_id order by count (domain) desc, adlist_id asc limit 1;"));

For more information:
  https://www.shellcheck.net/wiki/SC2128 -- Expanding an array without an ind...
  https://www.shellcheck.net/wiki/SC2207 -- Prefer mapfile or read -a to spli...
  https://www.shellcheck.net/wiki/SC2006 -- Use $(...) notation instead of le...
```

Fixing one by one led me to this… 🤷🏻‍♂️

Reply:
@yubiuser , could you please check your comment with my `shellcheck`?

Reply:
@yubiuser , had you time to review this?